<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>recordforsurg</class>
 <widget class="QDialog" name="recordforsurg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>400</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Gargi</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Запись на операцию</string>
  </property>
  <widget class="QLabel" name="lbStartTime_5">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>81</y>
     <width>250</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Gargi</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ФИО пациента</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="teTime">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>228</y>
     <width>350</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>350</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>350</width>
     <height>16777215</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="lbStartTime_6">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>130</y>
     <width>250</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Gargi</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ФИО назначающего врача</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbStartTime_7">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>277</y>
     <width>250</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Gargi</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Название операции</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbStartTime_8">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>12</y>
     <width>350</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>350</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>60</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Gargi</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Запись пациента на анализы</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QDateEdit" name="deDate">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>179</y>
     <width>350</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>350</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>350</width>
     <height>16777215</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="lbStartTime_4">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>179</y>
     <width>250</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Gargi</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Дата проведения операции </string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbPatient">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>81</y>
     <width>350</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>350</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>350</width>
     <height>16777215</height>
    </size>
   </property>
  </widget>
  <widget class="QComboBox" name="cbSurgery">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>277</y>
     <width>350</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>350</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>350</width>
     <height>16777215</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="lbStartTime_3">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>228</y>
     <width>250</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Gargi</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Время проведения операции </string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbDoctor">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>130</y>
     <width>350</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>350</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>350</width>
     <height>16777215</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAdd">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>326</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>465</x>
     <y>27</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
